java/test/jmri/jmrix/can/cbus/swing/nodeconfig/CbusNodeEditEventFrameTest.java

```diff
@@ -119,8 +112,9 @@ public void tearDown() {
         
         memo = null;
         tcis = null;
-        
+        super.tearDown(); 
         JUnitUtil.tearDown();
+       
```

Comment:
Take out the call to  JUnitUtil.tearDown.  It is being called in the super.tearDown method.

Reply:
Makes sense, done.